extensions/roulette/database/action.py

```diff
@@ -0,0 +1,23 @@
+from ..config import config
+from typing import Set
+
+
+class Role:
+    def __init__(self, id: int):
+        self.id = id
+
+
+def _get_timeout_roles(guild) -> Set[Role]:
+    role_ids_conf = config.timeout_roles()
+    role_ids = set()
+
+    for role_id_conf in role_ids_conf:
```

Comment:
`for role_id in role_ids_conf` (the iteration is over role IDs, not configurations). Maybe also make the set `timeout_roles` instead of `role_ids` for clarity